package.json

```diff
@@ -46,8 +45,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged",
-      "pre-push": "_scripts/audit.sh",
+      "pre-commit": "lint-staged && _scripts/git-precommit-hook.sh",
```

Comment:
Random question while I was trolling the Husky docs and .git/hooks/ directory... Do we want to use `pre-commit` or `pre-push`?
Not sure how [other] people use Git, but I know we've had variations of this argument over the past X years since some people commit a bunch of changes before pushing, and others it's more of a moot 1:1.

Reply:
Thanks Peter; yeah I talked about it with a couple of other folks (Dan and Barry). Our sense was that our workflows tend to be 1:1 between commits and pushes, so it wouldn't make a big difference.

That said, my current plan is to move this out of a git hook and make a separate, parallel PR CI task, so it likely won't matter for this PR after all.